assets/src/edit-story/panels/index.js

```diff
@@ -67,7 +71,11 @@ export function getPanels( elements ) {
 				case COLOR: return { type, Panel: ColorPanel };
 				case FONT: return { type, Panel: FontPanel };
 				case TEXT: return { type, Panel: TextPanel };
-				default: throw new Error( `Unknown panel: ${ type }` );
+				default: throw new Error( sprintf(
+					/* translators: %s: Type of panel. */
+					__( 'Unknown panel: %s', 'amp' ),
```

Comment:
Same here about developer-facing messaging. I'd not make that translatable

Reply:
I will remove.

Reply:
Removed in https://github.com/ampproject/amp-wp/pull/4143/commits/d4cd0a55ce8b81f0ad639c1f2c7a244444fafc34